releaseNotes/AzureSpringCloudFeatureManagement.md

```diff
@@ -0,0 +1,22 @@
+# azure-spring-cloud-feature-management
+
+[Source code][source_code] | [Package (Maven)][package] | [Samples][samples] | [Product documentation][docs]
+
+# azure-spring-cloud-feature-management-web
+
+[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Same Feature as `com.microsoft.azure` `azure-spring-cloud-feature-management` and `com.microsoft.azure` `azure-spring-cloud-feature-management-web`
+* Added Support for Feature Targeting
```

Comment:
While we call it "feature targeting" internally, customers will be lost. You probably can borrow some ideas that Jimmy had in the release notes for .NET feature management library.
https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/Microsoft.Featuremanagement.md#targeting